x-pack/plugins/observability_solution/synthetics/public/apps/embeddables/constants.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const STATUS_OVERVIEW_EMBEDDABLE = 'STATUS_OVERVIEW_EMBEDDABLE';
```

Comment:
@shahzad31 This is the name with which the embeddable gets registered to the embeddable registry. I recommend synthetics is include in the name, something like `SYNTHETICS_OVERVIEW_EMBEDDABLE`